201-vmaccess-on-ubuntu/azuredeploy.parameters.us.json

```diff
@@ -0,0 +1,36 @@
+{
```

Comment:
there doesn't appear to be anything here that is specific to US Gov cloud, so just remove the file.